Library/Homebrew/cask/lib/hbc/container.rb

```diff
@@ -53,12 +55,18 @@ def self.autodetect_containers
       # Hbc::Container::GenericUnar
     end
 
-    def self.for_path(path, command)
+    def self.for_path(path)
       odebug "Determining which containers to use based on filetype"
-      criteria = Criteria.new(path, command)
+
+      magic_number = if path.directory?
+        ""
+      else
+        File.binread(path, 262) || ""
```

Comment:
Would be good to make `262` a constant.

Reply:
It already is in `UnpackStrategy`, this is only temporary.

Reply:
@reitermarkus 👍